### DISCUSSION TITLE
[IntelliSense] v0.8.7 works, but versions above are non-functional

### DISCUSSION THREAD

**Discussion by stephanygrace** [tailwindlabs/tailwindcss#11587] [Help]
**What version of VS Code are you using?**

**_Version: 1.79.2_**

**What version of Tailwind CSS IntelliSense are you using?**

**_Only works in v0.8.7_**

**What version of Tailwind CSS are you using?**

**_"tailwindcss": "^3.3.2"_**

**What package manager are you using?**

**_npm_**

**What operating system are you using?**

**_Windows_**

**Tailwind config**

```js
module.exports = {
  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {
    extend: {},
  },
  plugins: [],
}

```

**VS Code settings**

```json
"tailwindCSS.includeLanguages": {
        "html": "html",
        "javascript": "javascript",
        "css": "css",
        "plaintext": "jsx"
    },
    "tailwindCSS.experimental.classRegex": [
        "class:\\s*\"([^\"]*)\""
    ],
    "tailwindCSS.emmetCompletions": true,
    "auto-rename-tag.activationOnLanguage": [
        "*"
    ],
```

**Describe your issue**

**_Versions of Tailwind CSS IntelliSense starting from v0.9.0 and above are no longer functioning; only the v0.8.7 and below are operational._**

**Comment by bradlc** [comment]
Hey @stephanygrace. I'm going to convert this to a discussion until we can identify an issue with the extension itself.

With the information you've provided it's going to be impossible to diagnose the issue. If you can provide the following information it will increase the chance of us being able to help:

- A GitHub repository containing a project where you are seeing this issue
- Logs from the language server output panel. You can reveal this using the `Tailwind CSS: Show Output` command
- Ensure that you have shared _all_ of your VS Code settings, including any workspace settings

Thanks!
